test/models/auditing/UserHasPenaltyAuditModelSpec.scala

```diff
@@ -369,7 +369,7 @@ class UserHasPenaltyAuditModelSpec extends SpecBase with LogCapturing {
         inactivePenaltyPoints = 2,
         regimeThreshold = 5,
         penaltyChargeAmount = 200,
-        PoCAchievementDate = LocalDate.of(2022, 1, 1)
+        PoCAchievementDate = Some(LocalDate.of(2022, 1, 1))
```

Comment:
Apparently, PoCAchievementDate is None when the `activePenaltyPoints = 0`, could you set this to None?

Reply:
changed